What's New: Restructure & Add Changelog (#8829)
DevExpress/devextreme-documentation: concepts/Common/Whats New/02 Changelog/002 UI and UX.md

---
title: UI/UX
---

### v26.1.3

> - We updated DevExtreme Fluent and Material themes to improve color-contrast accessibility of multiple DevExtreme components, such as:

Reviewer: ```suggestion
- We updated DevExtreme Fluent and Material themes to improve color-contrast accessibility of the following DevExtreme components:
```

    - CardView
    - DataGrid
    - Gantt
    - Scheduler
    - PivotGrid

    We also updated the base colors of Material Blue themes as follows:

    #include full-width-image-container with {
        src: "/images/Changelog/26.1.3/material-fluent-themes.png",
        alt: "A before and after of base colors used in DevExtreme Material Blue Light and Dark themes.",
        minWidth: "360px",
        width: "50vw"
    }

> - We updated the listening animation in our [SpeechToText](/Documentation/Guide/UI_Components/SpeechToText/Overview/) component: previously, the whole button flickered, indicating the listening process. Now, only the Stop icon flickers. [See Demo](https://js.devexpress.com/Demos/WidgetsGallery/Demo/SpeechToText/Overview/)

Reviewer: ```suggestion
- The listening animation in our [SpeechToText](/Documentation/Guide/UI_Components/SpeechToText/Overview/) component has also been updated: previously, the whole button flickered to indicate the listening process. Now, only the Stop icon flickers. [See Demo](https://js.devexpress.com/Demos/WidgetsGallery/Demo/SpeechToText/Overview/)
```

> - In jQuery [Popup](/Documentation/Guide/UI_Components/Popup/Overview/), the `.dx-template-wrapper` element is now rendered inside `.dx-popup-title` when using templates.

Reviewer: ```suggestion
- In jQuery [Popup](/Documentation/Guide/UI_Components/Popup/Overview/), the `.dx-template-wrapper` element is now rendered inside `.dx-popup-title` when templates are used.
```
